magenta/music/musicxml_parser.py

```diff
@@ -374,8 +379,13 @@ def __init__(self, xml_measure, state):
     self.tempos = []
     self.time_signature = None
```

Comment:
Would you like an exception to be raised whenever a second time signature in a measure is found? I'll also need to construct a test case for this.